packages/next/build/index.ts

```diff
@@ -809,7 +815,12 @@ export default async function build(dir: string, conf = null): Promise<void> {
           if (additionalSsgPaths.has(page)) {
             // TODO: maybe we should move it to a separate function? see duplicated code below
             const extraRoutes = additionalSsgPaths.get(page) || []
-            for (const route of extraRoutes) {
+
+            for (let route of extraRoutes) {
+              if (route === '/') {
+                route = 'index'
+              }
+
```

Comment:
@anulman I added extra code, so now it's not that easy to refactor this duplicated code